README.md

```diff
@@ -0,0 +1,361 @@
+    },
+    "pool_records": [
+      {
+        "deposit_msg_states": [],
+        "pool": {
+          "id": "1",
+          "pool_coin_denom": "pool96EF6EA6E5AC828ED87E8D07E7AE2A8180570ADD212117B2DA6F0B75D17A6295",
+          "reserve_account_address": "cosmos1jmhkafh94jpgakr735r70t32sxq9wzkayzs9we",
+          "reserve_coin_denoms": [
+            "uatom",
+            "uusd"
+          ],
+          "type_id": 1
+        },
+        "pool_batch": {
+          "begin_height": "563",
+          "deposit_msg_index": "2",
+          "executed": false,
+          "index": "3",
+          "pool_id": "1",
+          "swap_msg_index": "2",
+          "withdraw_msg_index": "2"
+        },
+        "pool_metadata": {
+          "pool_coin_total_supply": {
+            "amount": "1089899",
+            "denom": "pool96EF6EA6E5AC828ED87E8D07E7AE2A8180570ADD212117B2DA6F0B75D17A6295"
+          },
+          "pool_id": "1",
+          "reserve_coins": [
+            {
+              "amount": "1088843820",
+              "denom": "uatom"
+            },
+            {
+              "amount": "54551075322",
+              "denom": "uusd"
+            }
+          ]
+        },
+        "swap_msg_states": [],
+        "withdraw_msg_states": []
+      },
+      {
+        "deposit_msg_states": [],
+        "pool": {
+          "id": "2",
+          "pool_coin_denom": "poolA4648A10F8D43B8EE4D915A35CB292618215D9F60CE3E2E29216489CF1FAE049",
+          "reserve_account_address": "cosmos153jg5y8c6sacaexezk34ev5jvxpptk0kscrx0x",
+          "reserve_coin_denoms": [
+            "stake",
+            "uusd"
+          ],
+          "type_id": 1
+        },
+        "pool_batch": {
+          "begin_height": "0",
+          "deposit_msg_index": "1",
+          "executed": false,
+          "index": "1",
+          "pool_id": "2",
+          "swap_msg_index": "1",
+          "withdraw_msg_index": "1"
+        },
+        "pool_metadata": {
+          "pool_coin_total_supply": {
+            "amount": "1000000",
+            "denom": "poolA4648A10F8D43B8EE4D915A35CB292618215D9F60CE3E2E29216489CF1FAE049"
+          },
+          "pool_id": "2",
+          "reserve_coins": [
+            {
+              "amount": "10000000",
+              "denom": "stake"
+            },
+            {
+              "amount": "10000000",
+              "denom": "uusd"
+            }
+          ]
+        },
+        "swap_msg_states": [],
+        "withdraw_msg_states": []
+      }
+    ]
+  }
+}
+```
+
+### Protobuf and Swagger
+
+The API documentation for the liquidity module can be found on `http://localhost:1317/swagger-liquidity/` when you successfully boostrap your testnet in your local computer. Note that `swagger` config should be `true` in `$HOME/.liquidityapp/config/app.toml`. 
+You can also reference our [public swagger API documentation](https://app.swaggerhub.com/apis-docs/bharvest/cosmos-sdk_liquidity_module_rest_and_g_rpc_gateway_docs). 
```

Comment:
```suggestion
The API documentation for the liquidity module is available on `http://localhost:1317/swagger-liquidity/` after you successfully boostrap a testnet in your local computer. 

Note that `swagger` config must be `true` in `$HOME/.liquidityapp/config/app.toml`. 
The public Swagger API docs are also available on [Cosmos SDK Liquidity Module - REST and gRPC Gateway docs](https://app.swaggerhub.com/apis-docs/bharvest/cosmos-sdk_liquidity_module_rest_and_g_rpc_gateway_docs). 
```

Reply:
Tip: it's handy to use self describing links, per https://developers.google.com/style/link-text